docs_src/security/Ch-APIGateway.md

```diff
@@ -220,58 +236,44 @@ of [default service ports](../general/ServicePorts.md).
 
 ### Creating Access Token for API Gateway Authentication
 
-The API gateway is configured to require authentiation prior to
-passing a request to a back-end microservice.
+Authentication is more fully explained
+in the [authentication chapter](./Ch-Authenticating.md).
```

Comment:
Note that the EdgeX UI's login page gives a link to this chapter, hardcoded for v2.1: https://docs.edgexfoundry.org/2.1/security/Ch-APIGateway/#creating-access-token-for-api-gateway-authentication

That should perhaps be replaced with https://docs.edgexfoundry.org/3.0/security/Ch-Authenticating/

Reply:
https://github.com/edgexfoundry/edgex-ui-go/pull/598